### ISSUE TITLE
Gamehub is ignoring the path/s I've told it to install games to

### ISSUE THREAD

**Issue by awsdert** [tkashkin/GameHub#619]
Side note before the main problem, setting dialog is hiding the window buttons for closing etc, have to use Alt+F4 to close it.

I moved a folder that installation folder that ignored the path I had already given to the path I expected it to be in, I even corrected the paths prior to restarting gamehub, yet I still get "Install" appear instead of "Run", can you please fix this as it's pissing me off that I can't store my games on my dedicated drive (which is so that when I eventually fuck something up and have to re-install manjaro (Cinnamon x64 edition) I don't have to re-install the games, just restore a few permissions & maybe a few icons

![Screenshot from 2022-03-16 09-49-38](https://user-images.githubusercontent.com/7422982/158563829-a8857d49-8684-441a-9204-860a0a8a2540.png)

**Comment by JustinTJones** [comment]
I've noticed this issue too. My workaround is to symlink the directory GameHub uses to the directory I want it to use.
`ln -s ~/GOG Games /mnt/MEDIA/GOG/Games/linux` should do the trick (if `~/GOG Games` is the erroneously used directory - it's been a  while since I've done this on my system).

**Comment by awsdert** [comment]
Yeah I ended up doing the same thing, should not be needed though, defeats the point of providing the option to declare a specific path if it just ignores the path completely. I'd understand if it booted with the system and got it's search in prior to the drive being mounted but I don't have it set up for that so it should follow the path unless the path don't exist
